src/lib/cli/command.js

```diff
@@ -603,8 +603,7 @@ export default function ( opts ) {
 	if ( _opts.format ) {
 		args.option(
 			'format',
-			'Render output in a particular format. Accepts "table" (default), "csv", and "json".',
-			'table'
+			'Render output in a particular format. Accepts "table" (default), "csv", and "json".'
```

Comment:
Is it intentional that the way the default value is indicated here is different than how it's shown in `vip import media status`?

https://github.com/Automattic/vip-cli/pull/2054/files?w=1#diff-3f389e45eefc81ab62ddcb49277eb56a8189f924d70bc16a8ce1ebaa99eef412R51

Reply:
[Discussed in Slack](https://a8c.slack.com/archives/C07MM37JVGF/p1729009823069269). The supported options for this command could have been handled by `--format` at the time the command was introduced, but we will need to discuss this choice further in a BB8 rather than handling it here.